File: Iniciante/URI 1930.c
#include <stdio.h>
 
int main() { 
    int T1, T2, T3, T4, totalTomadas;

    scanf("%d %d %d %d", &T1, &T2, &T3, &T4);
    totalTomadas = (T1 + T2 + T3 + T4) - 3;
    printf("%d\n", totalTomadas);

    return 0;
}

File: Iniciante/URI 1930.php
<?php
    $line = explode(" ", trim(fgets(STDIN)));
    $T1 = intval($line[0]);
    $T2 = intval($line[1]);
    $T3 = intval($line[2]);
    $T4 = intval($line[3]);
    $totalTomadas = ($T1 + $T2 + $T3 + $T4) - 3;
    echo "$totalTomadas\n";
?>